qa-heal.planx-pla.net/dashboard/Public/documentation/index.html

```diff
@@ -276,7 +276,7 @@ <h6><a href="#Discovery">Discovery Page</a></h6>
                                     </div>
                                 </li>
                                 <li>
-                                    <a href="#Workspaces" style="background-image: url('img/workspace_flavors.png');"></a>
+                                    <a href="#Workspaces" style="background-image: url('img/workspaces_new_2.png');"></a>
```

Comment:
i don't see a `workspaces_new_2.png` file in the repo

Reply:
Added via PR 2094 that has been merged